### PR TITLE
#3180 update email addresses from github issue list

### LINKED ISSUE #3180: Change these example email addresses
Here is a list of email addresses we would like changed to `example@example.com`:

```
doesntmatter@nowhere.com
yourlogin@sendgrid.net
you@youreamil.com
person@youaremailing.com
alice@foo.com
casey@baz.com
john.doe@gmail.com
jane.doe@hotmail.com
your.email@example.com
replyto@example.com
teammate3@example.com
alice@sendgrid.com
example@juno.com
example2@band.la
example3@exampledomain.com
esting@sendgrid.ne
rawest@gmail.co
claims@msp.org
info@bnl.co.uk
richard@hengseng.com
test@brandonmwest.com
isaac@hotmail.comm
isaac@example.com
address@domain.com
grouch@trashcan.com
dominic.may@sendgrid.com
jones@example.com
jones2@example.com
e@example.com
example@example.com
customer@example.com
newcustomer@example.com
stomer@example.com
l1@domain.com
2@domain2.com
email1@domain.com
mail2@domain2.com
er@example.com
1@domain.com
xample@example.com
user2@example.c
le@example.com
user4@example.com
tomer@example.com
destination2@example.com
ion2@example.com
sampleuser@example.com
sampleemail@example.com
example2@yahoo.com
brandon.west@sendgrid.com
emailrecipient@domain.com
test@gmail.com
person@customer.com
dmarc.rua@customdomain.com
anything@parse.yourdomain.com
parse@parse.yourdomain.c
customersuccess@sendgrid.com
person@domain.coom
postmaster@thedomain.com
johndoe@gmail.com
johndoe1@gmail.com
postmaster@domain.com
4798911130a2069f3483fda756b8e81c@www.example.com
alex@email.sendgrid.biz
isaac@email.sendgrid.biz
```

## Expected content
All these emails should be removed and replaced with `example@example.com`

## Actual content

This is pretty pervasive throughout the docs content.

### PR DESCRIPTION
**Pull request by SimonSickle**
Done with sed
```
while read i; do
    LC_ALL=C find ./docs/ -type f -exec sed -i '' s##example@example.com#g {} +
done <emaillist.txt
```

Signed-off-by: Simon Sickle <simon@simonsickle.com>

**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #3180

@ksigler7

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/docs?pullRequest=3182) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/docs?pullRequest=3182) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/docs?pullRequest=3182) it.</sub>

File: source/API_Reference/SMTP_API/index.html [hunk 1]
```diff
@@ -105,7 +105,7 @@
 {% codeblock lang:json %}{
   "to": [
     "<ben@example.com>",
-    "Joe Smith <joe@example.com>"
+    "Joe Smith <joeexampexample@example.com>"
```
Comment (ksigler7) [inline source/API_Reference/SMTP_API/index.html]: example@example.com
example@example.com

Comment (PAPERPANKS) [reply source/API_Reference/SMTP_API/index.html]: I  have address this issue in my [commit](https://github.com/sendgrid/docs/pull/3206/commits/85181c5f86aa2015a66aecebdd1f6eeaa5b74224)

**Comment by thinkingserious** [comment]
Hello @SimonSickle,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**